### ISSUE TITLE
Ridge Racer Turbo, Screen Adjustment Lock up

### ISSUE THREAD

**Issue by HardwareBreaker** [MiSTer-devel/PSX_MiSTer#224]
I figured I should post this as a separate bug report just in case it got lost in the last one.

https://streamable.com/jgfas4

As you can see from the video adjusting the screen will eventually cause the game to lock up
and no controls respond. I do have vsync_adjust=2 enabled.

Maybe other games with screen adjust will have a similar issue?

**Comment by birdybro** [comment]
What happens when you set vsync_adjust=0?

**Comment by RobertPeip** [comment]
Yes, i could reproduce this. The game doesn't really hang, it still renders the image, but doesn't react to inputs anymore. No idea why.
